webapp/app/viewModels/role/roleList.html

```diff
@@ -2,13 +2,19 @@
     <table>
         <thead>
             <tr>
+                <th>Edit</th>
                 <th>Id</th>
                 <th>Rol Description</th>
                 <th>Created By</th>
+                <th>Action</th>
             </tr>
         </thead>
         <tbody data-bind="foreach : roles">
             <tr>
+                <td>
```

Comment:
What do you need this checkbox for?

Reply:
i have a idea for mark a rol for edit in the grid, change label for text, but in the talk i have with you , the edition are in the page of create, i add a button for ediiting and remove the checkbox